Anchal_Hora/nodejs/Login.js

```diff
@@ -0,0 +1,34 @@
+function func(){
+    window.location.href = "./admin/index.html";
+    }
+document.getElementById("button").addEventListener("click",Gettoken)
+function Gettoken(){
```

Comment:
use consistent casing of variables and function names, use all lower case

Reply:
> @horaanchal you need work on your code, you need to take care of your naming convention, for files, functions and variables. Please go through your whole code again and then raise a PR'
> Let me know if you have any doubts

Sure I will go through my code once again and let you know.

Reply:
> use consistent casing of variables and function names, use all lower case

Proper naming convention followed.